core/proto/keytransparency_v1_grpc/keytransparency_v1_admin_grpc.proto

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc";
+
+// Key Transparency Administration Service
+//
+// The Key Transparency Service API consists of a map of user names to public
+// keys. Each user name also has a history of public keys that have been
+// associated with it.
+package keytransparency.v1.grpc;
+
+import "proto/keytransparency_v1_proto/keytransparency_v1_admin_proto.proto";
+import "google/api/annotations.proto";
+
+// The KeyTransparencyAdminService API provides batch access to the directory of public keys. 
+service KeyTransparencyAdminService {
+  // BatchSetEntries uses an authorized_public key to perform a set request on multiple entries at once.  
+  rpc BatchUpdateEntries(keytransparency.v1.proto.BatchUpdateEntriesRequest) returns (keytransparency.v1.proto.BatchUpdateEntriesResponse) {
+    option (google.api.http) = {
+      post: "/v1/users:batch"
+      body: ""
+    };
+  }
+
+  rpc ListDomains(keytransparency.v1.proto.ListDomainsRequest) returns (keytransparency.v1.proto.ListDomainsResponse) {}
+
+  // GetDomain returns the confiuration information for a given domain.
+  rpc GetDomain(keytransparency.v1.proto.GetDomainRequest) returns (keytransparency.v1.proto.GetDomainResponse) {
+    option (google.api.http) = { get: "/v1/domains/{domain_id}" };
+  }
+
+  rpc CreateDomain(keytransparency.v1.proto.CreateDomainRequest) returns (keytransparency.v1.proto.CreateDomainResponse) {
```

Comment:
doc
Is there error behavior on this call? For example, trying to create an already-existing domain.

Reply:
done

Reply:
But do you think the errors are be well-known enough or obvious enough not to be worth mentioning in the API documentation?  Or is there some other place one would go to look for error documentation that applies to all of the API?